src/script/components/message/CallMessage.test.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import ko from 'knockout';
+import TestPage from 'Util/test/TestPage';
+import {CallMessage as CallMessageEntity} from 'src/script/entity/message/CallMessage';
+import CallMessage, {CallMessageProps} from './CallMessage';
+
+class CallMessagePage extends TestPage<CallMessageProps> {
+  constructor(props?: CallMessageProps) {
+    super(CallMessage, props);
+  }
+
+  getCallMessage = (completed?: string) =>
```

Comment:
```suggestion
  getCallMessage = (completed: 'completed' | 'not_completed') =>
```

I removed the optional because I don't see it being used without passing a string.

Reply:
The idea was to use the selector for 3 states. 
- Check that a call message in `completed` state exists
- Check that a call message in `not_completed` state exists
- And check that any call message exists

But can be change when needed.